### DISCUSSION TITLE
Not strictly a Junicode matter, but..

### DISCUSSION THREAD

**Discussion by afarlie** [psb1558/Junicode-font#160] [Ideas]
I note someone had extracted individual forms from an old work on court hand:-
https://commons.wikimedia.org/wiki/Category:Court_hand_letters_in_PNG

It may be possible for someone very technically competent to adapt them into a script font. ~~~~

**Comment by psb1558** [comment]
Whoever drew those had a very elegant hand. The complete book is [Andrew Wright, Court Hand Restored, 9th ed. (1879)](https://books.google.com/books/about/Court_hand_Restored.html?id=lXUKAAAAIAAJ). Full of good stuff.
